easybuild/easyconfigs/r/R/R-3.4.4-iomkl-2018a-X11-20180131.eb

```diff
@@ -0,0 +1,1775 @@
+    }),
+    ('mlogit', '0.2-4', {
+        'checksums': ['5d9f5d20efd9d654e15875d813d9da3e756c7b8f25d8bb1f5d689a128fa7cd96'],
+    }),
+    ('getopt', '1.20.2', {
+        'checksums': ['3d6c12d32d6cd4b2909be626e570e158b3ed960e4739510e3a251e7f172de38e'],
+    }),
+    ('gsalib', '2.1', {
+        'checksums': ['e1b23b986c18b89a94c58d9db45e552d1bce484300461803740dacdf7c937fcc'],
+    }),
+    ('optparse', '1.4.4', {
+        'checksums': ['0b1f1d336f1be386260fb5bbdf74f01787c09176d69febf8d563e5af75e5c4b9'],
+    }),
+    ('miniUI', '0.1.1', {
+        'checksums': ['9c92032071ee75ee9c26c4cfd3115877fbcb15de2fd2023109c670784b51cce1'],
+    }),
+    ('hms', '0.4.2', {
+        'checksums': ['a57820b3e3221e973cba9500b4ad7953730110ee398693d150af833f26d5d0bc'],
+    }),
+    ('readr', '1.1.1', {
+        'checksums': ['1a29b99009a06f2cee18d08bc6201fd4985b6d45c76cefca65084dcc1a2f7cb3'],
+    }),
+    ('forcats', '0.3.0', {
+        'checksums': ['95814610ec18b8a8830eba63751954387f9d21400d6ab40394ed0ff22c0cb657'],
+    }),
+    ('haven', '1.1.1', {
+        'checksums': ['5f7bf6dbee431b4e64771d9d97ee2c3da5b8f66a5e81fd1ec78a236eb3de6cba'],
+    }),
+    ('labelled', '1.0.1', {
+        'checksums': ['9b7f1c5644f0e30496c291c84c52204b47d2b7da9aaefa332898aaae3226bfa2'],
+    }),
+    ('classInt', '0.1-24', {
+        'checksums': ['f3dc9084450ea3da07e1ea5eeb097fd2fedc7e29e5d7794b418bcb438c4fcfa2'],
+    }),
+    ('questionr', '0.6.2', {
+        'checksums': ['1dcad3c980548352f3a775ec089359b3f7737762e97c2a97c9a39cda4de54f56'],
+    }),
+    ('klaR', '0.6-13', {
+        'checksums': ['56b9689fc7ed5ece159056bcea181c9e670205a570095e76779a2a03d156171b'],
+    }),
+    ('neuRosim', '0.2-12', {
+        'checksums': ['f4f718c7bea2f4b61a914023015f4c71312f8a180124dcbc2327b71b7be256c3'],
+    }),
+    ('locfit', '1.5-9.1', {
+        'checksums': ['f524148fdb29aac3a178618f88718d3d4ac91283014091aa11a01f1c70cd4e51'],
+    }),
+    ('GGally', '1.3.2', {
+        'checksums': ['f4143f45254fed794be991180aeffe459f6756bfa08acad963707d8e843cfd0a'],
+    }),
+    ('beanplot', '1.2', {
+        'checksums': ['49da299139a47171c5b4ccdea79ffbbc152894e05d552e676f135147c0c9b372'],
+    }),
+    ('clValid', '0.6-6', {
+        'checksums': ['bcf781238a2a831a7ec8909d27251731e3115a0541a9ea9795e5ec4d903938d1'],
+    }),
+    ('matrixStats', '0.53.1', {
+        'checksums': ['df70bc9fb64716a7a633b5d0c2a2422b2c8d93ac5e64574528a325505df9712e'],
+    }),
+    ('DiscriMiner', '0.1-29', {
+        'checksums': ['5aab7671086ef9940e030324651976456f0e84dab35edb7048693ade885228c6'],
+    }),
+    ('ellipse', '0.4.1', {
+        'checksums': ['1a9a9c52195b26c2b4d51ad159ab98aff7aa8ca25fdc6b2198818d1a0adb023d'],
+    }),
+    ('leaps', '3.0', {
+        'checksums': ['55a879cdad5a4c9bc3b5697dd4d364b3a094a49d8facb6692f5ce6af82adf285'],
+    }),
+    ('nloptr', '1.0.4', {
+        'checksums': ['0981a80c46c174a5458bdb8427b2db5428c3daca12ff2c30df7023f143fad7b3'],
```

Comment:
This is wrong somehow, I have `84225b993cb1ef7854edda9629858662cc8592b0d1344baadea4177486ece1eb` for a fresh download of `nloptr_1.0.4.tar.gz`, and also for my cached download which dates from Nov 17th...

Reply:
Yes, very strange. I deleted my tgz, and ran inject-checksums and now it's 84225b...